packages/authentication/src/__tests__/fixtures/services/jwt-service.ts

```diff
@@ -34,6 +34,7 @@ export class JWTService implements TokenService {
       // decode user profile from token
       userProfile = await verifyAsync(token, this.jwtSecret);
       userProfile[securityId] = userProfile.id;
+      delete userProfile.id;
```

Comment:
hmm, why have to delete the `id` field? I think its ok to preserve it.

Reply:
When someone changed the shopping cart application to include the [securityId] field work, a user profile is created without the `id` field.

e.g. https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/services/user-service.ts#L51

Reply:
I have re-instituted the `id` field in the user profile used in my mocha tests

Reply:
ah I see, thank you for the explanation.